conda/recipes/versions.yaml

```diff
@@ -71,7 +71,7 @@ fsspec_version:
 gdal_version:
   - '>=3.2.0,<3.3.0a0'
 geopandas_version:
-  - '>=0.6, <=0.8.1'
+  - '>=0.9.0'
```

Comment:
I believe in cuspatial we pinned it to `>=0.9.0,<0.10.0a0` since geopandas is not yet API stable.

Reply:
Thanks for pointing this out. In `cuspatial/meta.yaml` we're using as you specified, in `environments/cuspatial_dev_cuda*.yml` we're using what I specified. This is a discrepancy I've been seeing lately - shouldn't the conda dev environments use the same recipe as the conda release environment from `meta.yaml`? I'd like to get rid of the `.yml` files if possible.

Reply:
Also, fixed as you requested.